### PR TITLE
Always cleanup syncset, don't log error for oc delete

### PR DESCRIPTION
**Pull request by jewzaam**
Just annoying output at this time when running the script, this cleans up state in hive better and doesn't report useless errors in script output.

### CODE REVIEW

File: scripts/cluster-upgrade.sh [hunk 1]
```diff
@@ -143,13 +143,9 @@ EOF
...
             else
-                # cluster is already upgraded?
-                if [ -z $SS_VERSION_FROM ];
-                then
-                    # delete the syncset, it's not needed anymore
-                    oc -n $CD_NAMESPACE delete SyncSet $SS_NAME
-                    UPGRADE_DONE="${UPGRADE_DONE}  $CD_NAME ($OCP_CURRENT_VERSION)\n"
-                fi
+                # delete the syncset, it's not needed anymore
+                oc -n $CD_NAMESPACE delete SyncSet $SS_NAME 2>/dev/null
```
Comment (jharrington22) [inline scripts/cluster-upgrade.sh]: Quotes :slightly_smiling_face:

Comment (jharrington22) [reply scripts/cluster-upgrade.sh]: What are the errors? Can we suppress them in any other way that doesn't mask other potential unknown errors?

Comment (jewzaam) [inline scripts/cluster-upgrade.sh]:
```suggestion
                oc -n "$CD_NAMESPACE" delete SyncSet "$SS_NAME" 2>/dev/null
```

Comment (jewzaam) [reply scripts/cluster-upgrade.sh]: Error is the syncset doesn't exist.  We could do better error handling but my expectation is we'll throw this away some day so we can automate the upgrade process.  We can deal w/ fancy then.

**Pull request merged**